sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -125,7 +128,7 @@ trait CheckAnalysis extends PredicateHelper {
             }
 
           case s: SubqueryExpression =>
-            checkSubqueryExpression(operator, s)
+            checkAnalysisWithConf(s.plan, conf)
```

Comment:
This is wrong. Could you revert it back?

Reply:
Yup, sorry. Was too eager to merge back master